### ISSUE TITLE
Train my own dataset without void class

### ISSUE THREAD

**Issue by amokame** [TUI-NICR/ESANet#45]
Hello,
I want to train the model on my own dataset, which has no void class.
I changed the code in dataset_base.py and train.py as following.

dataset_base.py: 
![image](https://user-images.githubusercontent.com/44995377/188558182-96180db4-509e-4d57-a5a2-b55626f5c998.png)

train.py:
![image](https://user-images.githubusercontent.com/44995377/188558202-0f1b60b3-5126-4332-9194-ba9e40dd62d7.png)

The training code starts to run, but the mIoU for each epoch is very small even after 50 epoches. 
It seems to go wrong after my changes. 
Could you give me some advice about how to make it right?

Thank you.

**Comment by danielS91** [comment]
It looks like you are excluding class 0 (which is a valid class in your scenario) in your training loss function. You should double-check the confusion matrix as well.

**Comment by amokame** [comment]
Thanks for your advise!
Instead of changing the code, changing label to make it have the void class seems easier.
I tried and it worked.

**Comment by khy-111** [comment]
Hello！ I want to ask that how you make your format of own datasetbase into cityscapece's
 format
